Client/src/Utils/Theme/globalTheme.js

```diff
@@ -205,10 +206,9 @@ const baseTheme = (palette) => ({
 						borderRadius: theme.shape.borderRadius,
 					},
 					"& .MuiInputBase-input": {
-						height: "var(--env-var-height-2)",
```

Comment:
@marcelluscaio removing this breaks the `TextInput` component, what was removing this meant to solve?

![image](https://github.com/user-attachments/assets/5552aba5-b09a-4bd0-ae42-d12b9bd86918)